### ISSUE TITLE
Match is undefined

### ISSUE THREAD

**Issue by serggb** [LumaPictures/meteor-jquery-sparklines#2]
Hi,

can you check this error please?

![image](https://cloud.githubusercontent.com/assets/26628972/24254581/98546ebe-0fe3-11e7-9ac5-a3c0f95cb32a.png)

Thanks!

**Comment by tripflex** [comment]
Yup same error, flagged it on Atmosphere since this renders it basically unusable
